Remove the _config.rst file from installation_guide

willthames opened the pull request:
##### SUMMARY

The _config.rst file causes 163 duplicate label warning
messages - removing it and the link to it causes no
additional problems


##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
docs

##### ANSIBLE VERSION
<!--- Paste verbatim output from "ansible --version" between quotes below -->
```
ansible 2.6.0 (devel e10e0d42d8) last updated 2018/04/09 21:15:12 (GMT +1000)
  config file = None
  configured module search path = [u'/Users/will/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/will/src/ansible/lib/ansible
  executable location = /Users/will/src/ansible/bin/ansible
  python version = 2.7.14 (default, Mar  9 2018, 23:57:12) [GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.39.2)]
```


##### ADDITIONAL INFORMATION
Small sample of warning messages:

```
/Users/will/src/ansible/docs/docsite/rst/reference_appendices/config.rst:895: WARNING: duplicate label default_local_tmp, other instance in /Users/will/src/ansible/docs/docsite/rst/installation_guide/_config.rst
/Users/will/src/ansible/docs/docsite/rst/reference_appendices/config.rst:1734: WARNING: duplicate label inventory_ignore_patterns, other instance in /Users/will/src/ansible/docs/docsite/rst/installation_guide/_config.rst
/Users/will/src/ansible/docs/docsite/rst/reference_appendices/config.rst:1588: WARNING: duplicate label enable_task_debugger, other instance in /Users/will/src/ansible/docs/docsite/rst/installation_guide/_config.rst
/Users/will/src/ansible/docs/docsite/rst/reference_appendices/config.rst:1796: WARNING: duplicate label paramiko_host_key_auto_add, other instance in /Users/will/src/ansible/docs/docsite/rst/installation_guide/_config.rst
/Users/will/src/ansible/docs/docsite/rst/reference_appendices/config.rst:1818: WARNING: duplicate label persistent_command_timeout, other instance in /Users/will/src/ansible/docs/docsite/rst/installation_guide/_config.rst
```

dharmabumstead commented on docs/docsite/rst/installation_guide/intro_configuration.rst:
```diff
@@ -42,7 +42,7 @@ Environmental configuration
 If these environment variables are set, they will override any setting loaded from the configuration file.
 
-You can get a full listing of available environment variables from :doc:`_config`.
+You can get a full listing of available environment variables from :doc:`/reference_appendices/config`.
```
We're using refs now. Please add the following as the very first line of the file ../reference_appendices/config.rst:

.. _ansible_config:

[..rest of file...]

...and then change this line to: 

:ref:`ansible_config`

Thanks!

dharmabumstead reviewed (changes requested):
Thanks @willthames for the PR - please see review comment.

Commit:
Remove the _config.rst file from installation_guide

The _config.rst file causes 163 duplicate label warning
messages - removing it and the link to it causes no
additional problems

willthames commented:
@scottb ready_for_review (I used `configuration_settings` rather than `ansible_config` to reduce potential for confusion with `ansible-config` CLI page)

scottb commented:
@willthames -- you mentioned me, but I think you got the id wrong. I'm not involved in this project.

willthames commented:
Apologies @scottb, I meant to mention @dharmabumstead

acozine commented:
Hey @willthames thanks for submitting this - if I'd found it sooner I could have saved myself some work. Your suggestions were great, and we've incorporated the changes you to the Installation guide, so I'm going to close this PR. I appreciate your attention to detail and your interest in reducing the sphinx errors in the docs build!

willthames commented:
No worries @acozine - I'll know to mention you in future if I make docs fixes!
